src/Bicep.LangServer/Handlers/BicepSignatureHelpHandler.cs

```diff
@@ -236,7 +236,7 @@ private static SignatureInformation CreateSignature(FunctionOverload overload, I
             if (includeReturnType)
             {
                 typeSignature.Append(": ");
-                typeSignature.Append(overload.ReturnTypeBuilder(arguments));
+                typeSignature.Append(overload.TypeSignatureSymbol);
```

Comment:
>TypeSignatureSymbol [](http://example.com/codeflow?start=46&length=19)

Why this change?

Reply:
The thing was that when creating a method hover, the file content appeared in the function description as the return value. It was making signatures very hard to read. Therefore I introduced a concept of type signature, where when defining a function with dynamic type you also need to provide a constant type to be used in signatures.

Reply:
Makes sense.